linode_api4/objects/linode.py

```diff
@@ -1052,6 +1372,11 @@ def clone(
     def stats(self):
         """
         Returns the JSON stats for this Instance
+
+        API Documentation: https://api.linode.com/v4/linode/instances/{linodeId}/stats
+
+        :returns: The JSON stats for this Instance
+        :rtype: str
```

Comment:
I think the return value can be `dict`. Did you have a chance to call this function?